components/engine_traits/src/misc.rs

```diff
@@ -61,7 +61,19 @@ pub trait StatisticsReporter<T: ?Sized> {
     fn flush(&mut self);
 }
 
-#[derive(Default)]
+/// RocksDB example:
+/// lv0: f5: [{k1_t6, delete: ()}]
+/// lv1: f4: [{k1_t10, put: v13}, {k2_t11, put: tombstone}, {k3_t12, put: v32}]
```

Comment:
nit: maybe we could clarify the two kinds of "tombstone" a bit to reduce confusion?
```suggestion
/// lv0: f5: [{k1_t6, delete: () /* RocksDB tombstone */}]
/// lv1: f4: [{k1_t10, put: v13}, {k2_t11, put: tombstone /* MVCC delete */}, {k3_t12, put: v32}]
```